docs/javascript/README.md

```diff
@@ -12,418 +12,32 @@ You should be familiar with JavaScript, Node.js, and at least the basics mention
-
-Let see how to query for the latest 10 (sorted by timestamp) posts by a certain author:
-
-```javascript
-const SecretStack = require('secret-stack')
-const ssbKeys = require('ssb-keys')
-const caps = require('ssb-caps')
-const os = require('os')
-const path = require('path')
-const {and, type, author, paginate, descending, toCallback} = require('ssb-db2/operators')
-
-const sbot = SecretStack({ caps })
-  .use(require('ssb-db2'))
-  .call(null, {
-    path: path.join(os.homedir(), '.ssb'),
-    keys: ssbKeys.loadOrCreateSync(path.join(os.homedir(), '.ssb', 'secret'))
-  })
-
-sbot.db.query(
-  and(
-   type('post'),
-   author('@6CAxOI3f+LUOVrbAl0IemqiS7ATpQvr9Mdw9LC4+Uv0=.ed25519')
-  ),
-  descending(),
-  paginate(10),
-  toCallback((err, msgs) => {
-    console.log(msgs)
-    sbot.close()
-  })
-)
-// {
-//   results: [{
-//     key: '%eR84PjHsUuVu2AgQmKjsMru+O4tzr2LwB05RnobN1n8=.sha256',
-//     value: [Object],
-//     timestamp: 1613057102824
-//   }, ...],
-//   total: 1987,
-//   duration: 337
-// }
-```
-
-This query used two indexes: `post` and `author`. The first time you run this query, it will take a little longer because it needs to create the indexes first. This is where the JIT (just in time) comes into play. The way it works is that under the hood [`type`](https://github.com/ssb-ngi-pointer/ssb-db2/blob/580eaedef0d0696d9e279709147b69696710144e/operators/index.js#L27) uses a [seeker](https://github.com/ssb-ngi-pointer/ssb-db2/blob/580eaedef0d0696d9e279709147b69696710144e/seekers.js#L28) function to fetch out exactly the field of the message it is interested in while building the index. The result is then stored as a bit vector (1 or 0 if the input value was matched). Author is a [prefix index](https://github.com/ssb-ngi-pointer/jitdb#prefix-indexes) where one index can answer multiple values, while still returning a bit vector. These bit vector can then be combined using the operators `and` `or`.
-
-Tim Robinson has a [repo](https://github.com/timjrobinson/ssb-db2-examples) with a lot of great examples of queries using db2.
-
-Sometimes you want to query for something a little more complicated. This could be [mentions](https://github.com/ssb-ngi-pointer/ssb-db2#full-mentions) (an array properly on a message), for this purpose ssb-db2 has the ability to create level db backed indexes that can be combined with other jitdb indexes like this:
+## Plugins
 
-```javascript
-const SecretStack = require('secret-stack')
-const ssbKeys = require('ssb-keys')
-const caps = require('ssb-caps')
-const os = require('os')
-const path = require('path')
+Short of "secret-stack plugins" these are modules which extend the functionality of `secret-stack`
 
-const {and, type, author, paginate, descending, toCallback} = require('ssb-db2/operators')
+[](plugins.md ':include')
```

Comment:
pulled plug-ins out until storage file to make editing easier. 

they are then included so that the sidebar gives you nice nested contents nav